perf/kernbench.py.data/config

```diff
@@ -0,0 +1,2980 @@
+# Kernel Configuration
```

Comment:
@PraveenPenguin , I'll do that..I was just following the autotest method of kernbench. thanks for your comment.